YJE/Day12/실패율.js

```diff
@@ -0,0 +1,26 @@
+// 실패율 계산하기
+function failRate(N, stages, result) {
+    let totalPlayers = stages.length;
+
+    for (let stage = 1; stage <= N; stage++) {
+        let currentStageFail = stages.filter(s => s === stage).length;
+        let failRate = currentStageFail / totalPlayers;
+        result.push({ stage, failRate });
+        totalPlayers -= currentStageFail;
+    }
+}
+
+// 스테이지 번호를 실패율의 내림차순으로 정렬하기
+function sortResult(N, stages, result) {
+    result.sort((a, b) => {
+        return b.failRate - a.failRate;
+    });
+    return result.map(s => s.stage);
+}
```

Comment:
N과 stages는 지금 사용하지 않는 불필요한 인자입니다. N과 stages를 없애고 result만 받고 정렬, map처리를 해주면 좀 더 간결해진 코드가 될 것 같습니다.
```suggestion
function sortResult(result) {
    return result
        .sort((a, b) => b.failRate - a.failRate)
        .map(r => r.stage);
}
```